network/scapy/Makefile

```diff
@@ -0,0 +1,54 @@
+#
```

Comment:
Is there anything extra added in this scapy package? Otherwise, I would prefer to use the one from the OpenWRT feeds.

Thanks!

Reply:
OpenWRT seems to have an older version (2.3.1-1).
I've just been trying to keep it updated, as quite a few changes and improvements have been made to it since then.

Changelog:
https://github.com/secdev/scapy/releases